src/main/resources/assets/api-docs.yaml

```diff
@@ -2565,6 +2635,31 @@ components:
             email: email authorization
             profile: profile authorization
   schemas:
+    Acknowledgement:
+      type: object
+      properties:
+        userId:
+          type: integer
+          description: The User ID for the current user
+        ack_key:
+          type: string
+          description: The name of the key
+        first_acknowledged:
+          type: integer
+          description: The date first acknowledged, in seconds since January 1, 1970
+        last_acknowledged:
+          type: integer
+          description: The date last acknowledged, in seconds since January 1, 1970
```

Comment:
should be `ackKey`, `firstAcknowledged`, etc. since they've been updated

Reply:
Nice reminder.  I'm used to two modes of using swagger - and I appreciate your keen eye while I train my mind around this one.